bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/html_basic/internal/RenderKitUtil.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2000-2017 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.bridge.renderkit.html_basic.internal;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import javax.faces.component.UIComponent;
+import javax.faces.context.FacesContext;
+
+
+/**
+ * @author  Kyle Stiemann
+ */
+public final class RenderKitUtil {
+
+	// Package-Private Constants
+	/* package-private */ static final String HEAD_RESOURCES_TO_RENDER_IN_BODY = "headResourcesToRenderInBody";
+	/* package-private */ static final String SCRIPT_RENDERER_TYPE = "javax.faces.resource.Script";
+	/* package-private */ static final String STYLE_CLASS_PORTLET_BODY = "liferay-faces-bridge-body";
+	/* package-private */ static final String STYLESHEET_RENDERER_TYPE = "javax.faces.resource.Stylesheet";
+
+	private RenderKitUtil() {
+		throw new AssertionError();
+	}
+
+	/* package-private */ static Set<String> getHeadResourceIds(FacesContext facesContext) {
```

Comment:
@ngriffin7a,
Not really, this class could either be made `/* package-private */` or if you'd rather make it `public`, you could move it to the `util.internal` package and make all those constants and methods public. It probably doesn't matter since this is all in `internal` packages.